pages/dkp/konvoy/1.8/release-notes/kubernetes-base-addon/index.md

```diff
@@ -14,6 +14,25 @@ enterprise: false
 
 For instructions on how to apply KBA updates, see [Introduction to KBAs](../../addons)
 
+February 17, 2022
+
+[stable-1.20-4.4.0](https://github.com/mesosphere/kubernetes-base-addons/releases/tag/stable-1.20-4.3.0)
+
+-   kube-oidc-proxy:
+    - Bumps kube-oidc-proxy to 0.3.0 to resolve "kubectl log" latency issues (COPS-7123)
+
+-   Nvidia
+    - Fixes Nvidia GPU platform service placement constraints (COPS-7142)
+    - Updates Nvidia DCGM exporter to 2.2.9 to fix metrics (COPS-7132)
+
+-   Prometheus
+    - Alertmanager CRD is now properly upgraded (COPS-6842)
+    - Adds concurrency policy Replace configuration to the Grafana home dashboard CronJob to avoid potentially creating an unbound number of pods (COPS-7105)
```

Comment:
```suggestion
    - A concurrency policy has been added to the Grafana dashboard CronJob to prevent the unintended creation of an unbound number of pods. (COPS-7105)
```